content/blog/mirai-2-5-0/index.Rmd

```diff
@@ -0,0 +1,164 @@
+    parallel and distributed computing to the ecosystem.
+photo:
+  url: https://unsplash.com/photos/a-bunch-of-different-colored-sashes-hanging-on-a-wall-OEiN_lSyQqE
+  author: Matt Benson
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: [mirai, parallelism]
+---
+
+<!--
+TODO:
+* [x] Look over / edit the post's title in the yaml
+* [x] Edit (or delete) the description; note this appears in the Twitter card
+* [x] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [x] Find photo & update yaml metadata
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [x] `usethis::use_tidy_thanks()`
+-->
+
+We're excited to announce [mirai](https://mirai.r-lib.org) 2.5.0, bringing production-grade async computing to R!
+
+This milestone release delivers enhanced observability through OpenTelemetry, reproducible parallel RNG, and key user interface improvements. We've also packed in twice as many [changes](https://mirai.r-lib.org/news/index.html) as usual - going all out in delivering a round of quality-of-life fixes to make your use of mirai even smoother!
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("mirai")
+```
+
+## Introduction to mirai
+
+mirai (Japanese for 'future') provides a clean, modern approach to parallel computing in R.
+Built on current communication technologies, it delivers extreme performance through professional-grade scheduling and an event-driven architecture.
+
+It continues to evolve as the foundation for asynchronous and parallel computing across the R ecosystem, powering everything from [async Shiny](https://rstudio.github.io/promises/articles/promises_04_mirai.html) applications to [parallel map](https://www.tidyverse.org/blog/2025/07/purrr-1-1-0-parallel/) in purrr to [hyperparameter tuning](https://tune.tidymodels.org/news/index.html#parallel-processing-2-0-0) in tidymodels.
+
+```{r}
+library(mirai)
+
+# Set up persistent background processes
+daemons(4)
+
+# Async evaluation - non-blocking
+m <- mirai({
+  Sys.sleep(1)
+  100 + 42
+})
+m
+
+# Results are available when ready
+m[]
+
+# Shut down persistent background processes
+daemons(0)
+```
+
+## A unique design philosophy
+
+### Modern foundation
+
+mirai builds on [nanonext](https://nanonext.r-lib.org), the R binding to Nanomsg Next Generation, a high-performance messaging library designed for distributed systems. This means that it's using the very latest technologies, and supports the most optimal connections out of the box: IPC (inter-process communications), TCP or secure TLS. It also extends base R's serialization mechanism to support custom serialization of newer cross-language data formats such as safetensors, Arrow and Polars.
+
+### Extreme performance
```

Comment:
I'd suggest making this a bulleted list like

* **Extreme performance**:  As a consequence of its solid technological foundation...
* **Production first**: ...
* **Deploy everywhere**: ...